evap/evaluation/models.py

```diff
@@ -525,19 +533,18 @@ def grades_activated(self):
     def update_courses(cls):
         logger.info("update_courses called. Processing courses now.")
         from evap.evaluation.tools import send_publish_notifications
-        today = datetime.date.today()
 
         courses_new_in_evaluation = []
         evaluation_results_courses = []
 
         for course in cls.objects.all():
             try:
-                if course.state == "approved" and course.vote_start_date <= today:
+                if course.state == "approved" and course.vote_start_date.date() <= datetime.date.today():
```

Comment:
now that's the one place where we actually need to use the time and not just the date, isn't it?